test/mysql/8.4-conf.d/build.cnf

```diff
@@ -0,0 +1,21 @@
+# This MySQL configuration file is mounted into the Database container (/etc/mysql/conf.d)
```

Comment:
Probably just nit, but I would structure this as `test/mysql/conf.d/{MYSQL_VERSION}/build.cnf` then mount `test/mysql/conf.d/{MYSQL_VERSION}`